### DISCUSSION TITLE
Can't create journals & notes

### DISCUSSION THREAD

**Discussion by parasiteoflife** [TechbeeAT/jtxBoard#1577] [Q&A]
I'm using DAVx5 and Nextcloud. I've tried creating a new calendar using DAVx5 enabling all three to no avail.

The collections are shown in the "Collections" section but when I try to create a journal or note I can only pick "Local" and there is nothing else listed.
First I couldn't even create a task but after creating a test note and a test task on Nextcloud directly I can finally create tasks and the test task is shown in jtx Board but the test note is not shown and as I said I can't create notes with the Nextcloud collection.

I'm using jtx Board 2.09.01.ose.

**Comment by patrickunterwegs** [comment]
Hi @parasiteoflife , could you please check if you can find the reason for the problem here: 
https://jtx.techbee.at/troubles-with-the-synchronization-see-what-could-go-wrong

**Reply by parasiteoflife** [reply]
Yes, I have done it all. I must say that when I create a new calendar in DAVx5 I only see "calendar" and "tasks" in what I suppose is what the collection support even though I've marked the 3 options when creating the calendar, but it doesn't matter because unless I create a test task in Nextcloud directly, jtx won't show the new collection when creating a task.

**Comment by thewholelifetolearn** [comment]
I encounter the same problem. I can't create a journal, a note or a task on a distant server. Only on local. I tried on several servers (Nextcloud and Roundcube) but it doesn't change anything. 
Even if there is data on the distant feeds, I can't choose them. 
I just displays them.

**Reply by patrickunterwegs** [reply]
Hi @parasiteoflife  ,what app version, android version and phone do you use?

**Reply by thewholelifetolearn** [reply]
Sorry, I forgot to add the information. 
I'm on android 13, Samsung phone and using 2.11.02.gplay of jtx Boards

**Comment by rogercreagh** [comment]
?DavX setup incorrect? Is DavX5 syncing journals and tasks to jtxB ok. You need to enable it per calendar'collection.

Nextcloud doesn't (or didn't until recently, but the feature might still be in beta) create Calendars with VJOURNAL enabled - for an existing calendar you have to go into the database and manually add VJOURNAL (using phpmyadmin or similar)

If you create a new calendar through DavX (assuming you've got that configured correctly for jtxB) then it will create the calendar with VJOURNAL enabled if you tell it to - Nextcloud does allow that. But for a preexisting calendar DavX/jtxB  can't add journal or task support remotely to Nextcloud, it has to happen when the calendar entry in the DB is first created.

It may be that most recent version of NC has finally fixed this, check what version your server is running.

**Reply by thewholelifetolearn** [reply]
Sorry I took some to answer. I took time for me to understand your comment.

Yes, my davx setup is correct and the collections are shown in jtx.
![Screenshot_20250507_202556_jtx Board](https://github.com/user-attachments/assets/0c5dbbdb-128a-4ac8-8d91-b28d465e9349)
![Screenshot_20250507_202509_jtx Board](https://github.com/user-attachments/assets/750e03da-5998-4cbd-a9b7-a4f913b15318)
![Screenshot_20250507_202117_DAVx](https://github.com/user-attachments/assets/fc6c81a9-6638-4cc0-8571-6b081a1e455c)
![Screenshot_20250507_201859_DAVx](https://github.com/user-attachments/assets/4e0a57c7-6d95-49aa-aa0e-6c9805b9fa3e)

As you can see, my nextcloud instance has journal activ. They are native. I didn't go through davx. 
On Cpanel, there is no note, but with the others are available.

**Comment by bvdbos** [comment]
There's [https://apps.nextcloud.com/apps/journalnotes](https://apps.nextcloud.com/apps/journalnotes) now and I created a caldav-bridge : [https://apps.nextcloud.com/apps/journal_caldav](https://apps.nextcloud.com/apps/journal_caldav).

Also, I created a caldav-bridge for Nextcloud Notes : [https://apps.nextcloud.com/apps/notes_caldav](https://apps.nextcloud.com/apps/notes_caldav).
